buildbot/env_config.py

```diff
@@ -0,0 +1,221 @@
+"""
```

Comment:
Will this script work outside intel network? It should!!!

Reply:
No, it can not work outside intel network. This script helps to set up the testing environment by parsing the dependency.conf file. The config file "dependency.conf" depends on runtimes on share folder in intel network, so this script also depends on share folder in intel network.
For outside intel network, there is no unified runtimes installation path. So it is difficult to set up the environment.

Reply:
If it does not work there is no reason to have it in intel/llvm

Reply:
Ok, I will close it.